array/fixedarray.mbt

```diff
@@ -29,7 +29,10 @@
 /// ```
 /// TODO: change the intrinsic to match the function name
 #intrinsic("%fixedarray.iter")
-pub fn[T] FixedArray::each(self : FixedArray[T], f : (T) -> Unit) -> Unit {
+pub fn[T] FixedArray::each(
+  self : FixedArray[T],
+  f : (T) -> Unit?Error
+) -> Unit?Error {
   for v in self {
```

Comment:
this is a big change that some changes to function with `intrinsic` should be carefuly examined, can you split it into small PRs?  cc @Yu-zh